source/extensions/filters/listener/proxy_protocol/proxy_protocol.cc

```diff
@@ -77,11 +80,20 @@ Network::FilterStatus Filter::onAccept(Network::ListenerFilterCallbacks& cb) {
   return Network::FilterStatus::StopIteration;
 }
 
+ReadOrParseState Filter::resetAndContinue(Network::IoHandle& io_handle) {
+  // Release the file event so that we do not interfere with the connection read events.
+  io_handle.resetFileEvents();
+  cb_->continueFilterChain(true);
+  return ReadOrParseState::Done;
```

Comment:
This method doesn't need to return anything, and the returned status should probably be only at the end of `Filter::onReadWorker()`

Reply:
agree, addressed in https://github.com/envoyproxy/envoy/pull/18951/commits/2b4a89a9fb037465fbd322a116e140d54837ea2e